compiler/luci/import/src/CircleReader.cpp

```diff
@@ -66,10 +66,14 @@ const char *tensor_name(const circle::Tensor *tensor)
   assert(tensor != nullptr);
 
   static const char *kEmptyTensorName = "(noname)";
-  const auto tensor_name = tensor->name()->c_str();
 
-  if (!std::string(tensor_name).empty())
-    return tensor_name;
+  if (tensor->name() != nullptr)
```

Comment:
Why did you change this? name shouldn't be null.

Reply:
The original model I experimented on had no names, so I added these changes and forgot to remove them from the draft. I have removed these changes.